introduction.md

```diff
@@ -0,0 +1 @@
+"To subdue the enemy without fighting is the acme of skill." - Sun Zu
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.